scripts/check_catalog_schema.py

```diff
@@ -39,6 +40,19 @@ def main(directory: Path) -> int:
             )
             code = 1
 
+        extra_fields = get_extra_fields(schema.field_names)
+
+        if len(extra_fields):
+            for extra_field in extra_fields:
+                field = schema.get_field(extra_field)
+                if not (field.type == "boolean" or field.type == "string"):
```

Comment:
(Nit) Parfois j'utilise cette autre écriture (qui serait difficile à faire en JS), qui est peut-être plus lisible :

```suggestion
                if field.type not in ("boolean", "string"):
```